astropy/uncertainty/core.py

```diff
@@ -304,7 +304,7 @@ def __repr__(self):
             reprarr = reprarr[firstspace+1:-1]  # :-1] removes the ending '>'
             return '<{} {} with n_samples={}>'.format(self.__class__.__name__,
                                                       reprarr, self.n_samples)
-        else:  # numpy array-like
+        else:  # numpy array_like
```

Comment:
Wait a minute... This is not part of a docstring. Too zealous with the `sed`? 😉 
```suggestion
        else:  # numpy array-like
```